assets/app/features/upgrade/steps.factory.spec.js

```diff
@@ -50,13 +50,541 @@ describe('Stepes Factory', function () {
             active: false,
             finished: false
         }
-    ];
+        ],
+        initialStatusData = {
```

Comment:
Follow up card: https://trello.com/c/ZOJZdSuH/2120-research-options-for-efficient-unit-testing-of-responses-from-status-api